total_tolles_ferleihsystem/src/app/shared/rest/api-base.service.ts

```diff
@@ -101,30 +136,14 @@ export class BaseApiService {
 
         const request = new AsyncSubject<T>();
         this.runningRequests.set(url, request);
-        this.http.get<T>(url, options).pipe(
-            catchError((error: any) => {
-                this.runningRequests.delete(url as string);
-                if (error.status != null) {
-                    return observableThrowError({status: error.status,
-                        message: (error._body.startsWith != null && error._body.startsWith('{')) ?
-                                JSON.parse(error._body).message : error.status + ' Server error'});
-                }
-                return observableThrowError(error.json().error || 'Server error');
-            }),
-        ).subscribe((res) => {
+        this.http.get<T>(url, options).subscribe((res) => {
             request.next(res);
             request.complete();
             this.runningRequests.delete(url as string);
         }, (error: any) => {
-            if (error.status != null) {
-                request.error({
-                    status: error.status,
-                    message: (error._body.startsWith != null && error._body.startsWith('{')) ? JSON.parse(error._body).message : error.status + ' Server error'}
-                );
-            } else {
-                request.error(error.json().error || 'Server error');
-            }
             this.runningRequests.delete(url as string);
+            console.log(error);
```

Comment:
remove debug statement (or change to console.warn or .error)
console.log should only be used in development as it corresponds to an info logging level that would spam the normal console in production. If this is intended as an error log in production it should use the appropriate method or be marked with a comment.